apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -0,0 +1,745 @@
+        initialized();
+
+        require(_minAcceptQuorumPct <= _supportRequiredPct, ERROR_INIT_PCTS);
+        require(_supportRequiredPct < PCT_BASE, ERROR_INIT_SUPPORT_TOO_BIG);
+
+        token = _token;
+        supportRequiredPct = _supportRequiredPct;
+        minAcceptQuorumPct = _minAcceptQuorumPct;
+        voteTime = _voteTime;
+    }
+
+    /**
+    * @notice Change required support to `@formatPct(_supportRequiredPct)`%
+    * @param _supportRequiredPct New required support
+    */
+    function changeSupportRequiredPct(uint64 _supportRequiredPct)
+        external
+        authP(MODIFY_SUPPORT_ROLE, arr(uint256(_supportRequiredPct), uint256(supportRequiredPct)))
+    {
+        require(minAcceptQuorumPct <= _supportRequiredPct, ERROR_CHANGE_SUPPORT_PCTS);
+        require(_supportRequiredPct < PCT_BASE, ERROR_CHANGE_SUPPORT_TOO_BIG);
+        supportRequiredPct = _supportRequiredPct;
+        emit ChangeSupportRequired(_supportRequiredPct);
+    }
+
+    /**
+    * @notice Change minimum acceptance quorum to `@formatPct(_minAcceptQuorumPct)`%
+    * @param _minAcceptQuorumPct New acceptance quorum
+    */
+    function changeMinAcceptQuorumPct(uint64 _minAcceptQuorumPct)
+        external
+        authP(MODIFY_QUORUM_ROLE, arr(uint256(_minAcceptQuorumPct), uint256(minAcceptQuorumPct)))
+    {
+        require(_minAcceptQuorumPct <= supportRequiredPct, ERROR_CHANGE_QUORUM_PCTS);
+        minAcceptQuorumPct = _minAcceptQuorumPct;
+        emit ChangeMinQuorum(_minAcceptQuorumPct);
+    }
+
+    /**
+    * @notice Change overrule window to a duration of `@transformTime(_newOverruleWindow)`
+    * @param _newOverruleWindow New overrule window in seconds
+    */
+    function changeOverruleWindow(uint64 _newOverruleWindow)
+        external
+        authP(MODIFY_OVERRULE_WINDOW_ROLE, arr(uint256(_newOverruleWindow), uint256(overruleWindow)))
+    {
+        require(_newOverruleWindow <= voteTime, ERROR_INVALID_OVERRULE_WINDOW);
+        overruleWindow = _newOverruleWindow;
+        emit ChangeOverruleWindow(_newOverruleWindow);
+    }
+
+    /**
+    * @notice `_representative == 0x0 ? 'Set your voting representative to ' + _representative : 'Remove your representative'`
+    * @param _representative Address of the representative to be allowed on behalf of the sender. Use the zero address for none.
+    */
+    function setRepresentative(address _representative) external isInitialized {
+        representatives[msg.sender] = _representative;
+        emit ChangeRepresentative(msg.sender, _representative);
+    }
+
+    /**
+    * @notice Create a new vote about "`_metadata`"
+    * @param _executionScript EVM script to be executed on approval
+    * @param _metadata Vote metadata
+    * @param _castVote Whether to also cast newly created vote
+    * @return voteId id for newly created vote
+    */
+    function newVote(bytes _executionScript, string _metadata, bool _castVote)
```

Comment:
Not sure if we want to support the `castVote` mechanism here, or if `forward()` should do the same.

I think we shouldn't (especially on forward, as that was only meant for early execution).